octoprint_mrbeam/iobeam/iobeam_handler.py

```diff
@@ -199,10 +202,9 @@ def _on_mrbeam_plugin_initialized(self, event, payload):
 
         self._subscribe()
 
-        # We only start the iobeam listener now
-        iobeam_worker = threading.Timer(1.0, self._initWorker, [self._socket_file])
-        iobeam_worker.daemon = True
-        iobeam_worker.start()
+        self._worker = threading.Thread(target=self._work, name="iobeamHandler")
+        self._worker.daemon = True
+        self._worker.start()
```

Comment:
starting the iobeam worker does not have a consequence as it will be waiting for the socket file to show up.